kura/org.eclipse.kura.wire.component.provider/src/main/java/org/eclipse/kura/internal/wire/subscriber/CloudSubscriberOptions.java

```diff
@@ -66,8 +66,7 @@
     int getSubscribingQos() {
         int subscribingQos = DEFAULT_QOS;
         final Object qos = this.properties.get(CONF_QOS);
```

Comment:
Since the configuration-policy is "require", properties will never be null.
I can agree with you that a best practice will be to have a check for properties==null but, as was the code before, if properties was null, every get method would end up with a NullPointer due to the this.properties.get call before properties!= null check the following line.